event_manager.py

```diff
@@ -20,9 +20,28 @@
 
 import json
 import re
+import os
+import logging
+from logging.handlers import RotatingFileHandler
 from flask import Flask, request, make_response
 from bot import KarmaBot
 
+# The "primary" logger is for the Flask app.  Both event_manager and the KarmaBot have their own
```

Comment:
But why? Couldn't we have them all log into the same logfile and just have a log formatter that shows the class/file that a log message is coming from.

Reply:
All levels of the logger already do log to the same file-- the ".bot" strings etc are UIDs for each level of the logger, not a filename.